spec/board_spec.rb

```diff
@@ -1,20 +1,39 @@
 require_relative "../lib/board"
 
 describe "Board" do
+  subject(:board) { Board.new }
+  subject(:test_player1) { TestPlayer.new("Z") }
+  subject(:test_player2) { TestPlayer.new("Y") }
   context "place_mark" do
```

Comment:
I believe typically you have one subject in a test.  You could have a let block instead for TestPlayer.

Reply:
Ok, will make that change.